aiida/backends/djsite/db/migrations/0024_dblog_update.py

```diff
@@ -38,6 +38,13 @@ def get_unknown_entity_log_no(dblog_model):
         objname__startswith=leg_workflow_prefix).count()
 
 
+def get_logs_with_no_nodes_no(dblog_model, dbnode_model):
```

Comment:
Maybe it would be more clear to change the `_no` suffix to `_number` or `get_number_logs_with_no_nodes` because the `no_nodes_no` is a bit confusing